v5/heroku.go

```diff
@@ -1617,31 +1623,6 @@ func (s *Service) BuildDeleteCache(ctx context.Context, appIdentity string) (*Bu
 	return &build, s.Delete(ctx, &build, fmt.Sprintf("/apps/%v/build-cache", appIdentity))
 }
 
-// A build result contains the output from a build.
-type BuildResult struct {
```

Comment:
This schema drops things, which would be a major version, breaking change for the public API of this client.

Reply:
Yeah, I was hoping for just a one liner but it gave me a lot more.

Reply:
Circling back to this, even after porting your `generate` changes from #47, it still deletes this `struct` @mars. I'm guessing this the reflected change from the schema?

Reply:
After researching this internally, those Platform API changes are removing things that have been marked deprecated for a year, and promoting Enterprise Team APIs to GA.

So, while unexpected, this is normal and folks seem to be of the opinion to not make this a major release (to `v6`).

Reply:
Sounds good mars.